src/engine-actions/create-channels.js

```diff
@@ -0,0 +1,186 @@
+const {
+  connectPeer,
+  openChannel
+} = require('../lnd-actions')
+const {
+  Big,
+  networkAddressFormatter,
+  loggablePubKey
+} = require('../utils')
+const getUncommittedBalance = require('./get-uncommitted-balance')
+const { CHANNEL_ROUNDING } = require('../constants')
+
+/**
+ * Determine the amount to fund channels with by taking into account
+ * uneconomic channels (that are too small to be worth it.)
+ * @param   {Engine} engine
+ * @param   {string} fundingAmount - int64 string of amount desired to open
+ * @param   {number} roundBehavior - Behavior to use when encoutnering a channel that is too small.
+ * @returns {Big}                    Modified amount to open in channels
```

Comment:
I'd assume we're okay returning `Big` since this is a private method?

Reply:
Yes that's the idea.